node/services/service.go

```diff
@@ -212,3 +220,36 @@ func FullAvailability(
 	})
 	return ca
 }
+
+// FraudedLifecycle wraps service starting.
+// It allows to check if Fraud Proof was stored in the local storage before service starts.
+// If there are no Fraud Proofs, then startF will be invoked.
+func FraudedLifecycle(
```

Comment:
❤️ 

The naming might be a bit off, but I don't have any better alternatives

Reply:
`FraudAlertLifecycle` ? I also don't know.

Reply:
Why not simply "FraudLifecycle"